cluster/images/provider-upjet-alibabacloud/Makefile

```diff
@@ -36,7 +48,39 @@ img.build.shared:
 		-t $(IMAGE) \
 		$(IMAGE_TEMP_DIR) || $(FAIL)
 
+img.publish: batch-process
+
 img.promote:
 	@$(INFO) Skipping image promotion from $(FROM_IMAGE) to $(TO_IMAGE)
 	@echo Promote is deferred to xpkg machinery
 	@$(OK) Image promotion skipped for $(FROM_IMAGE) to $(TO_IMAGE)
+
+CONCURRENCY ?= 30
+DEP_CONSTRAINT := >= 0.0.0
+ifeq (-,$(findstring -,$(VERSION)))
+    DEP_CONSTRAINT = >= 0.0.0-0
+endif
+BUILD_ONLY ?= false
+STORE_PACKAGES ?= ""
+batch-process: $(UP)
```

Comment:
we're actively removing dependencies on the `up` CLI in c-c repos, so this has been ported over to `upjet`.

See [this](https://github.com/jbw976/provider-upjet-aws/commit/d518261e66fb115f92710dabb3d6b0a66314e4a8#diff-3a8d10e40868dd87f4b16b5f61e2f6a7fc3bc7a12693d21675f735bc6dc32af4) commit as an example